ui_framework/components/global_styles/variables/_z_index.scss

```diff
@@ -0,0 +1,5 @@
+$kuiZContent:         1;
+$kuiZContentMenu:     100;
+$kuiZNavigation:      200;
+$kuiZMask:            300;
+$kuiZModal:           400;
```

Comment:
I did some research and I think we're OK to use much much larger numbers. Looks like the max is 2147483647 in most browsers, and 16777271 in Safari 3:

* http://puidokas.com/examples/z-index_max/
* https://www.quora.com/Is-there-a-maximum-value-for-z-index
* https://stackoverflow.com/questions/491052/minimum-and-maximum-value-of-z-index